lib/Doctrine/Import/Builder.php

```diff
@@ -179,6 +179,14 @@ class Doctrine_Import_Builder extends Doctrine_Builder
      */
     protected $_phpDocEmail = '##EMAIL##';
 
+
+    /**
+     * Contains the actAs columns after running buildSetUp
+     *
+     * @var array
```

Comment:
Can we be more precise here? Maybe `array<string, array{name: string, type: string, ?disabled: bool}>` can do? Wdyt?